### PR TITLE
Allow to show Zhuyin and/or Pinyin or none

### PR DESCRIPTION
**Pull request by jiru**
This PR allows to selectively show either Pinyin, either Zhuyin, or none of them. This is useful for learners who don’t want to see pinyin or don’t want to see any transliteration at all.

### CODE REVIEW

**Comment by paulvi** [comment]
What is Zhuyin ? Is it https://en.wikipedia.org/wiki/Bopomofo ? 注音符號

I guess this PR would be could for Chinese, who already know (almost) all characters.  
Yet default should stay the current

**Comment by jiru** [comment]
> What is Zhuyin ? Is it https://en.wikipedia.org/wiki/Bopomofo ? 注音符號

Yes.

> I guess this PR would be could for Chinese, who already know (almost) all characters.

Yes, it could be useful for native Chinese speakers, as well as learners of Chinese using Bopomofo instead of Pinyin.

> Yet default should stay the current.

It does in my implementation. The default is to show Pinyin only.

**Comment by leezu** [comment]
Hi @jiru, please excuse the late response. @cschiller has refactored the code of the plugin and we now have a shared codebase for the Chrome and Firefox version. Could you rebase your commit on the new version?

File: js/options.js [hunk 1]
```diff
@@ -45,10 +45,14 @@ function restoreOptions() {
...
-      document.optform.zhuyin[1].selected = true;
+    if (options.translit == 'zhuyin') {
+      document.optform.translit[1].selected = true;
+    } else if (options.translit == 'pinyin') {
+      document.optform.translit[2].selected = true;
+    } else if (options.translit == 'zhuyin_pinyin') {
+      document.optform.translit[3].selected = true;
     } else {
-      document.optform.zhuyin[0].selected = true;
+      document.optform.translit[0].selected = true;
```
Comment (cschiller) [inline js/options.js]: This effectively makes "none" the default if options.translit is not set. The default should be pinyin.

**Pull request closed without merging**